### ISSUE TITLE
Row and column headings in tables

### ISSUE THREAD

**Issue by nordstromjf** [PreTeXtBook/pretext#1091]
For accessibility and visual purposes, there should be an ability to add a heading (label?) for the rows and for the columns. For example, rows labeled by Player 1, columns by Player 2. It also might be good for accessibility reasons to have each individual column/row label to have a tag, to indicate they aren't cells of the table. In the example below, H and T are really labels, not cells.
Sample table as it is done now:
<tabular>
          <row>
           <cell></cell>
           <cell></cell>
           <cell colspan="2" halign="center">Player 2</cell>
          </row>
          <row>
            <cell></cell>
            <cell></cell>
            <cell bottom="minor" halign="center">H</cell>
            <cell bottom="minor" halign="center">T</cell>
          </row>
          <row>
            <cell>Player 1</cell>
            <cell right="minor">H</cell>
            <cell bottom="minor" right="minor"><m>(1, -1)</m></cell>
            <cell bottom="minor" right="minor"><m>(-1, 1)</m></cell>
          </row>
          <row>
            <cell></cell>
            <cell right="minor">T</cell>
            <cell bottom="minor" right="minor"><m>(-1, 1)</m></cell>
            <cell bottom="minor" right="minor"><m>(1, -1)</m></cell>
          </row>
      </tabular>
    </table>

**Comment by Alex-Jordan** [comment]
I'm not sure that the example is rendering as intended here in the GitHub post. Is there a link to a live version we could look at?

Column and row headings have always been in the roadmap. One problem is where to draw the line with what we will bend over backwards to support or not. For example, two rows of column headers, where the headers in the first row span multiple columns? And if so, should the markup be layout markup, or somehow capture that the header cells in the second row are "children" of the multicolumn header cells in the first row? And other complications.

**Comment by nordstromjf** [comment]
I think column and row headers would be sufficient. Here is the live link:https://nordstromjf.github.io/IntroGameTheory/S_IntroMatrices.html

**Comment by smborio** [comment]
Anyone knows if in this headers one could define the text to appear vertically? In the first table in the live link I would want for example the Head and Tail to appear vertically. Is this something that is possible already or would it need to be added here?

**Comment by rbeezer** [comment]
Column headings (only), but including vertical text.  At 888973021f9b9eb5f0bb548e7ebe07ae7bd5920e and subsequent.  Announcement on `pretext-dev`.

**Comment by rbeezer** [comment]
Just a note to myself - leaving this open until column headings get done.

**Comment by Alex-Jordan** [comment]
Row headers are implemented, but the attribute `@header` on a `row` throws a schema validation complaint. Is that an oversight or is it intentional to wait on adding it to the schema?

**Comment by rbeezer** [comment]
Not yet addressed, for no reason other than a desire to get a good night's rest once in a while.  ;-)

**Comment by Alex-Jordan** [comment]
Is there no objection if I add it then with a PR? I'm happy to. I like my jing reports clean :)

**Comment by rbeezer** [comment]
Go for it.

**Comment by rbeezer** [comment]
Implemented starting at 7c77b95f102539e9c1f9e630945a5f2ae63b6f69

See -announce and -dev for more.

Thanks, @nordstromjf , for your patience!
